third_party/nvfuser/test/test_expr_simplifier.cpp

```diff
@@ -47,7 +47,9 @@ void assertSimplifiedMod(Val* x, Val* y, Val* z) {
 
 } // namespace
 
-TEST_F(NVFuserTest, FusionAssociativeAndCommutativeReordering_CUDA) {
```

Comment:
We were using this naming convention during the time when we link our tests with other jit tests. Now our tests has a standalone executable, so I don't think these naming conventions provide us with any benefit any more. So I am removing it to get a shorter test name.

Reply:
Checked with @jjsjann123 offline, we should keep the `_CUDA` suffix, but we can change the remaining. Updated this PR.